sentinel-adapter/sentinel-grpc-adapter/src/test/java/com/alibaba/csp/sentinel/adapter/grpc/GrpcTestServer.java

```diff
@@ -0,0 +1,32 @@
+package com.alibaba.csp.sentinel.adapter.grpc;
+
+import io.grpc.Server;
+import io.grpc.ServerBuilder;
+
+import java.io.IOException;
+
+class GrpcTestServer {
+    private Server server;
+
+    GrpcTestServer() {
+    }
+
+    void start(int port, boolean shouldintercept) throws IOException {
+        if (server != null) {
+            throw new IllegalStateException("Server already running!");
+        }
+        ServerBuilder<?> serverBuild = ServerBuilder.forPort(port)
+                .addService(new FooServiceImpl());
+        if(shouldintercept)
```

Comment:
Please reformat your code with [Alibaba P3C Plugin](https://github.com/alibaba/p3c) and obey the rules in Alibaba Java Coding Guidelines (naming, indent, format, etc.). For example:

```java
if (shouldIntercept) {
    serverBuilder.intercept(new SentinelGrpcServerInterceptor());
}
```